doc/neps/nep-0052-python-api-cleanup.rst

```diff
@@ -214,12 +274,15 @@ To discuss:
   and are too easy to shoot yourself in the foot with.
 
 
+.. S: consider `np.dtypes`.
```

Comment:
Hmm interesting thought. Maybe - would be good to first decide on what to remove though, and then see what is left? We don't need 3 or 4 aliases for each dtype.

Reply:
Right, I was mainly concerned that the new submodule names could be confusing, so wanted to find something that explicitly explains their intent.